File: app/src/main/java/me/eigenein/nexttrainwear/fragments/StationsFragment.java
package me.eigenein.nexttrainwear.fragments;

import android.app.Fragment;
import android.os.Bundle;
import android.support.wearable.view.WearableRecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import me.eigenein.nexttrainwear.Preferences;
import me.eigenein.nexttrainwear.R;
import me.eigenein.nexttrainwear.StationCatalogue;
import me.eigenein.nexttrainwear.adapters.StationsAdapter;

public class StationsFragment extends Fragment {

    private WearableRecyclerView recyclerView;

    public StationsFragment() {
        // Do nothing.
    }

    public static StationsFragment newInstance() {
        return new StationsFragment();
    }

    @Override
    public View onCreateView(
        final LayoutInflater inflater,
        final ViewGroup container,
        final Bundle savedInstanceState
    ) {
        recyclerView = (WearableRecyclerView)inflater.inflate(R.layout.fragment_stations, container, false);
        return recyclerView;
    }

    @Override
    public void onStart() {
        super.onStart();
        recyclerView.setAdapter(new StationsAdapter(Preferences.getStations(getActivity())));
    }
}

File: app/src/main/java/me/eigenein/nexttrainwear/fragments/StationsFragment.kt
package me.eigenein.nexttrainwear.fragments

import android.app.Fragment
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import me.eigenein.nexttrainwear.R
import me.eigenein.nexttrainwear.adapters.StationsAdapter
import me.eigenein.nexttrainwear.utils.getStations
import me.eigenein.nexttrainwear.utils.wearableRecyclerView
import org.jetbrains.anko.*

class StationsFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return UI {
            wearableRecyclerView(R.style.VerticalRecyclerView) {
                layoutParams = ViewGroup.LayoutParams(matchParent, matchParent)
                horizontalPadding = dip(20)
                verticalPadding = dip(60)
                clipToPadding = false
                layoutManager = LinearLayoutManager(activity)
                adapter = StationsAdapter(activity.getStations().toMutableSet())
            }
        }.view
    }
}
